subprojects/docs/src/snippets/scala/crossCompilation/groovy/build.gradle

```diff
@@ -15,30 +15,8 @@ dependencies {
 
 // tag::scala-cross-compilation[]
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_6
-}
-
-assert hasProperty('java6Home') : "Set the property 'java6Home' in your your gradle.properties pointing to a Java 6 installation"
-def javaExecutablesPath = new File(java6Home, 'bin')
-def javaExecutables = [:].withDefault { execName ->
-    def executable = new File(javaExecutablesPath, execName)
-    assert executable.exists() : "There is no ${execName} executable in ${javaExecutablesPath}"
-    executable
-}
-
-tasks.withType(AbstractCompile) {
```

Comment:
Because the initial sample does not do what it says, this ends up being equivalent.
Not sure if properly fixing the sample is a goal.